src/pages/ApplyMain/index.js

```diff
@@ -15,14 +15,27 @@ const ApplyMainPage = () => {
   const [innerWidth, setInnerWidth] = useState(window.innerWidth);
   const DESKTOP_WIDTH = 1199;
   const TABLET_WIDTH = 599;
+  const [clickedArray, setClickedArray] = useState([false, false, false]);
+
+  const handleClick = index => {
+    let newArr = [...clickedArray];
+    if (newArr[index] === true) {
```

Comment:
이제 다시 보니 여기는 줄여서 쓸 수 있을 것 같아요
```suggestion
    if (newArr[index]) {
```

Reply:
저는 왜 이런 쉥각을 못할ㄲㅏ요????